content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -299,95 +378,30 @@ those identities.
     caption="Multiple service meshes with certificate authorities"
     >}}
 
-## Mesh models
+## Isolating teams within the mesh
```

Comment:
I think we should talk about multi-mesh here too, so re-title as Isolating teams with Istio or something.

Reply:
Isn't this just a rename of `Tenancy Models`? I'm not sure this is an improvement ... or am I missing something?